tests/unit_tests/test_time_series.py

```diff
@@ -292,6 +293,28 @@ def test_check_unknown_resolution_pass():
         assert check_resolution(time_series) is None
 
 
+def test_check_rate_is_not_zero_pass():
+    time_series = pynwb.TimeSeries(name="test", unit="test_units", data=[1, 2, 3], rate=4.0)
+    assert check_rate_is_not_zero(time_series) is None
+
+
+def test_check_rate_is_not_zero_single_frame_pass():
+    time_series = pynwb.TimeSeries(name="test", unit="test_units", data=[1], rate=0.0)
+    assert check_rate_is_not_zero(time_series) is None
+
+
+def test_check_rate_is_not_zero_timestamp_is_none_fail():
+    time_series = pynwb.TimeSeries(name="test", unit="test_units", data=[1, 2, 3], rate=0.0)
+    assert check_rate_is_not_zero(time_series) == InspectorMessage(
+        message=f"{time_series.name} has a sampling rate value of 0.0Hz but the series has more than one frame.",
```

Comment:
Minor suggestion

Lately we've been in the habit of making tests as explicit as possible to help identify potential issues quicker - a part of that means avoiding f-strings where possible / not inconvenient

```suggestion
    time_series = pynwb.TimeSeries(name="TimeSeriesTestt", unit="n.a.", data=[1, 2, 3], rate=0.0)
    assert check_rate_is_not_zero(time_series) == InspectorMessage(
        message="TimeSeriesTest has a sampling rate value of 0.0Hz but the series has more than one frame.",
```

Reply:
Ok! thanks for the correction